app.py

```diff
@@ -368,6 +369,16 @@ def parse():
     username = current_user()['username'] or None
     exception = None
 
+    user = DATABASE.user_by_username(username)
```

Comment:
This way we make an additional database call on each `/parse` request. It would be nice to store the `test_group` value on the user (in the `session` for example). If it is already there, we don't have to check it each time. As we parse quite a lot this will introduce a lot of db calls.

Reply:
First of all @TiBiBa, thanks a lot for your helpful suggestions!

We have discussed a few ways to fix the problem: 
1. we can link the test_group field to the session user only, so no real database query is needed.
2. we move the check if the test_group field exists to the /login endpoint (login via login page as well as login via session cookie).

If we decide to persist the test_group field in the database in its own table, can you give us some guidance here on how to create our own table and attach it to the user? A link to a doc or tutorial would be perfect as we are not that familiar with this yet.

Reply:
I would like this to be checked at the `/login` endpoint. Also automatically assigning a user to a group if it isn't in one yet. I would be happy to help! Please send me a message in Discord (if your already in there?) to get in touch, I can give you some pointers on adding a table to our database.